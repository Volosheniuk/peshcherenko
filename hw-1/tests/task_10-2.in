a 100 3.1415 3.14159264
