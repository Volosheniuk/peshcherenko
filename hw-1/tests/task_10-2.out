a
100
3.141500
3.141593
